src/bundle/DependencyInjection/Configuration/Parser/Notifications.php

```diff
@@ -17,12 +17,14 @@
  *
  * Example configuration:
  * ```yaml
- * ezpublish:
+ * ibexa:
  *   system:
  *      admin_group: # configuration per siteaccess or siteaccess group
  *          notifications:
  *              warning: # type of notification
  *                  timeout: 5000 # in milliseconds
+ *          notification_count:
+ *              interval: 60000 # in milliseconds
```

Comment:
I would have prefere the following but couldn't do it, see https://github.com/ibexa/admin-ui/pull/1198/files#r1523327082
```yaml
notifications:
    warning: # type of notification
        timeout: 5000 # in milliseconds
    count:
        interval: 60000 # in milliseconds
```

Reply:
That won't work, as notifications are not queried by type. You either have a notification, or you do not.

Reply:
> That won't work, as notifications are not queried by type. You either have a notification, or you do not.

@Steveb-p Not sure about this remark. My indentation doesn't set `count` for a `type` but for all types by being directly below `notifications`. Which can't be done actually because the config parser says that everything below `notifications` is a `type`.

Reply:
My remark is about what you suggested in the comment :sweat_smile: 

I'm just saying that we're doing ajax call in the background to get all notifications, and the primary reason to introduce this is to reduce the unnecessary ajax calls. Adding type-specific interval would mean more ajax calls (we need to respect the setting somehow, right?), so there would be no gain :sweat_smile:

Reply:
After discussions, it seems that there can be a confusion between `notifications.warning.timeout` being about a notification bar and `notifications.count.interval` being about custom notification messages.

At least, `notification_count.interval` dissociate the two features.

Reply:
Yeah... this was a bad idea to put notification types directly under `notifications` key. I think we can clean that up, put custom types under `notifications.types` and introduce normalization to provide backwards compatibility with old config. Let me know if you want to try to do it yourself or someone will help you out.

Reply:
@webhdx I would prefer to leave this whole PR to someone else. My initial goal was more to expose the issue and proof it could be enhanced, not really to solve it.